bindings/gcp/bucket/metadata.yaml

```diff
@@ -0,0 +1,106 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: gcp.bucket
+version: v1
+status: alpha
+title: "GCP Storage Bucket"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/gcpbucket/
+binding:
+  output: true
+  operations:
+    - name: create
+      description: "Create an item."
+capabilities: []
+authenticationProfiles:
+  - title: "GCP API call Authentication"
```

Comment:
Please make this a shared auth profile too, like we're doing for AWS.

Reply:
I've submitted a shared auth profile as part of this PR.